src/user.ts

```diff
@@ -188,14 +188,25 @@ export class UserModule {
      * @param bootstrapData - data was got from bootstrap resource
      */
     public getAccountInfoInBootstrap(bootstrapData: any) {
-        const { bootstrapResource } = bootstrapData;
+        const {
+            bootstrapResource: {
+                accountSetting: {
+                    login,
+                    firstName,
+                    lastName,
+                    links: { self },
```

Comment:
We can use alias as `links: { self: profileUri },`

Reply:
fixed